### ISSUE TITLE
Suggestion: hold to hold

### ISSUE THREAD

**Issue by GijsTimmers** [zond/android-diplicity#87]
Would it be possible to issue a hold command by just holding the unit? Seems intuitive to me.

**Comment by zond** [comment]
No order is already equivalent to a hold order..

**Comment by GijsTimmers** [comment]
that's true but to me it would seem as a nice addition. guess we have a difference of opinion here.

**Comment by zond** [comment]
I agree it's nice! But it doesn't STOP you from coding ;)

**Comment by GijsTimmers** [comment]
Yes, that's definitely true.
